BGAnimations/ScreenSelectProfile underlay/Input.lua

```diff
@@ -12,6 +12,16 @@ local profile_data = args.ProfileData
 -- ScreenSelectProfile's code across multiple files.
 local finished = false
 
+-- Counter used to determine if both players have selected their profile. 
+-- This value basically represents the amount of players that are ready to
+-- move forward + 1. Each time a player presses enter this value goes up until
+-- it matches the amount of human players
+local playersSelected = 1
```

Comment:
I feel a little inclined to have this be a table with just the current state:
```lua
{
  ["P1"] = false,
  ["P2"] = false,
}
```
Default it to true if the player is not joined and then just check if both are true to determine to move to the next screen or not.

Reply:
How's this?